src/components/ConversationCard/ConversationCard.jsx

```diff
@@ -147,7 +147,7 @@ const ConversationCards = () => {
               <p className="ml-[2px] text-[#60605F] font-[600]">4</p>
             </div>
             <div
-              className="gap-[.55rem] h-[100%] max-[732px]:px-[4px] bg-white rounded-[8px]
+              className="h-full max-[732px]:px-[4px] bg-white rounded-[8px]
```

Comment:
don't delete gap-[.55rem]

Reply:
2 ulitity classes are put together. Which is bad practice n results in performance issues later. Therefore removed the 1st once because it was getting overrided by 2nd gap class. 

If it was intention to put gap-[0.55rem] n not the other one. Then why put it first? It was anyhow not getting applied

Reply:
delete gap-[4px]